src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.ts

```diff
@@ -32,6 +32,8 @@ export class ExamParticipationCoverComponent implements OnChanges, OnDestroy, On
     @Input() submitInProgress = false;
     @Input() attendanceChecked = false;
     @Input() testRunStartTime: dayjs.Dayjs | undefined;
+    @Input() isProduction = true;
+    @Input() isTestServer = false;
```

Comment:
Since the footer height is different in production and testservers and the height of the cover is based on the footer height, we need to differentiate between test servers and production.